poketechnika/Camera.h

```diff
@@ -1,14 +1,16 @@
 #pragma once
-#include "GameObject.h"
 #include <SFML/Graphics.hpp>
+#include "GameObject.h"
+#include "Map.h"
+#include "InputHandler.h"
 
 class Camera : public GameObject
 {
 	static sf::Vector2f size;
+	sf::Vector2f position;
```

Comment:
dlaczego nie wrzucić tego bespośrednio do GO, skoro każdy obiekt będzie to miał, nie tylko te drawable?

Reply:
Racja, jak najbardziej można tak zrobić

Reply:
Chociaż z drugiej strony - czy GO które nie jest Drawable potrzebuje position (poza szczególnym przypadkiem kamery?)

Reply:
ponownie powołując się na unity - nawet jak czegoś nie widać, ma swoją pozycję (czasami się to przydaje, kiedy jest jakiś niewidzialny obiekt który w jakiś sposób jest mechanicznie potrzebny, ale nie jest widoczny). W naszym przypadku, jak mówiłem w tym głównym komentarzu, raczej whatever, to jest tak raczej w szerszej skali.